From 6244e550cb65d34698676d2dc16f7af8a9b783f1 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Tue, 29 Apr 2014 17:07:38 -0400
Subject: [drm] tegra: Include header drm/drm.h

Message-id: <1398791260-24689-2-git-send-email-prarit@redhat.com>
Patchwork-id: 79228
O-Subject: [RHEL7.1 PATCH BZ 1090078 1/3] drm/tegra: Include header drm/drm.h
Bugzilla: 1090078
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Rob Clark <rclark@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Rik van Riel <riel@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1090078

commit a191e48d442f2d996e2a7292802a2ad22ecb503a
Author: Emil Goode <emilgoode@gmail.com>
Date:   Fri Apr 26 19:49:51 2013 +0200

    drm/tegra: Include header drm/drm.h

    Include definitions of used types by including drm/drm.h

    Sparse output:
    /usr/include/drm/tegra_drm.h:21:
     found __[us]{8,16,32,64} type without
     #include <linux/types.h>

    Signed-off-by: Emil Goode <emilgoode@gmail.com>
    Signed-off-by: Thierry Reding <thierry.reding@gmail.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/uapi/drm/tegra_drm.h b/include/uapi/drm/tegra_drm.h
index 6e132a2..73bde4e 100644
--- a/include/uapi/drm/tegra_drm.h
+++ b/include/uapi/drm/tegra_drm.h
@@ -17,6 +17,8 @@
 #ifndef _UAPI_TEGRA_DRM_H_
 #define _UAPI_TEGRA_DRM_H_
 
+#include <drm/drm.h>
+
 struct drm_tegra_gem_create {
  __u64 size;
  __u32 flags;
-- 
1.7.1